post_installation_configuration/coreos-layering.adoc

```diff
@@ -40,31 +57,78 @@ Some Hotfixes require a Red Hat Support Exception and are outside of the normal
 +
 In the event you want a Hotfix, it will be provided to you based on link:https://access.redhat.com/solutions/2996001[Red Hat Hotfix policy]. Apply it on top of the base image and test that new custom layered image in a non-production environment. When you are satisfied that the custom layered image is safe to use in production, you can roll it out on your own schedule to specific node pools. For any reason, you can easily roll back the custom layered image and return to using the default {op-system}.
 +
-.Example Containerfile to apply a Hotfix
+.Example on-cluster Containerfile to apply a Hotfix
 [source,yaml]
 ----
 # Using a 4.12.0 image
-FROM quay.io/openshift-release-dev/ocp-release@sha256...
-#Install hotfix rpm
-RUN rpm-ostree override replace https://example.com/myrepo/haproxy-1.0.16-5.el8.src.rpm && \
-    rpm-ostree cleanup -m && \
-    ostree container commit
+containerfileArch: noarch
+content: |-
+  FROM configs AS final
+  #Install hotfix rpm
+  RUN rpm-ostree override replace https://example.com/myrepo/haproxy-1.0.16-5.el8.src.rpm && \
+      rpm-ostree cleanup -m && \
+      ostree container commit
+----
++
+.Example out-of-cluster Containerfile to apply a Hotfix
+[source,yaml]
+----
+# Using a 4.12.0 image
+containerfileArch: noarch
+content: |-
+  FROM quay.io/openshift-release-dev/ocp-release@sha256...
```

Comment:
issue: I don't think that this would work like this. Or at the very least, if we instruct people to do it this way, we'll need to keep in mind that their final image would not have the MachineConfigs baked into it. That's why one uses the `FROM configs AS final` line; so that they can get the image that we've already layered things onto.

For the out-of-cluster scenario, they still need to set the `osImageURL` field on a MachineConfig for the time-being.